frontend/src/components/molecules/DetailView.tsx

```diff
@@ -7,9 +8,59 @@ import {
   getDevelopmentArea,
   getTradeOff,
 } from '../../api/ontologies';
+import { mapCorrelationToName } from '../../common/node';
 import { RootState } from '../../state/store';
 import { Annotation, Node } from '../../types/ontologyTypes';
 import Connections from '../atoms/Connections';
+import SlideInDrawer from '../atoms/SlideInDrawer';
+
+interface ContextDividerProps {
```

Comment:
We have used types for Props instead of interfaces previously in the app. I can't see anywhere that inherits from these props. Is there any reason for using interface?